docs/tasks.md

```diff
@@ -5,6 +5,7 @@ Tasks enable file-centric workflows in Box. User can create tasks on files and a
 
 * [Create a Task](#create-a-task)
 * [Get a Task's Information](#get-a-tasks-information)
+* [Delete a Task Permanently](#delete-a-task-permanently)
```

Comment:
All other functions which move items to trash  are called delete and those who removed items permanently are called deletePermanently.  I wanted to be consistent so user accidentally doesn't use deleteTask thinking it works same as other delete functions. It might clear for people familiar with sdk but not to new people. If you want to rename it I would at least improve documentation.